docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -176,6 +176,23 @@ For each Trino table which maps to BigQuery view there exists a system table whi
 Given a BigQuery view ``customer_view`` you can send query
 ``SELECT * customer_view$view_definition`` to see the SQL which defines view in BigQuery.
 
+.. _bigquery-sql-support:
+
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in
```

Comment:
We may want to avoid mentioning "write" access because INSERT and CTAS statement aren't yet supported.

Reply:
We've been using "read and write access to data and metadata" as a boilerplate for connectors, even when there are heavy limitations on write access such as not supporting INSERT. I'd much prefer to keep that original text.